sci-biology/fsl/fsl-5.0.8.ebuild

```diff
@@ -133,7 +133,9 @@ src_install() {
 	dosym /etc /usr/share/fsl/etc
 	dosym /usr/share/doc/${P} /usr/share/fsl/doc
 
-	doenvd "${FILESDIR}"/99fsl
+	cp "${FILESDIR}"/99fsl "${TMPDIR}"/99fsl || die
+	eprefixify "${TMPDIR}"/99fsl
+	doenvd "${TMPDIR}"/99fsl	
 	mv "${ED}"/usr/bin/{,fsl_}cluster || die
```

Comment:
This can be simplified to a single line with prefixify_ro from prefix.eclass, e.g.

`doenvd "$(prefixify_ro "${FILESDIR}"/99fsl)"`